frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -24,7 +24,8 @@ class UserSuccessModal extends React.Component {
     const { onClose, user, temporaryPassword } = this.props;
     return (
       <ModalContent
-        title={t`${user.getName()} has been added`}
+        // https://user-images.githubusercontent.com/1937582/172612281-9cfd67a5-b228-4176-a64b-5ee3e2369179.png
+        title={t`${user.common_name} has been added`}
```

Comment:
User's [`getName`](https://github.com/metabase/metabase/pull/23111/files#diff-4f46204c4fd684bc8b51d8c6f9a2d791ce065787b6de8dce6808772d1d02f5c8L33) is used sparingly, so it would be better that we drop this usage as most user props are just plain objects anyway.

<img src="https://user-images.githubusercontent.com/1937582/172612281-9cfd67a5-b228-4176-a64b-5ee3e2369179.png" />